3scale/porta
# THREESCALE-10153: Do not sync apps with Zync when not using OpenID Connect

**What this PR does / why we need it**:

Currently, information is being sync'ed with Zync that is not really relevant and does not bring any value (just creates more load). This PR skips synchronization for applications that are using API Key or App ID / App Key authentication.

**Which issue(s) this PR fixes** 

[THREESCALE-10153](https://issues.redhat.com/browse/THREESCALE-10153)

**Verification steps** 

- Create/delete/update an application in API Key or App ID/App Key mode
- Verify that Zync is not notified

**Special notes for your reviewer**:
